### ISSUE TITLE
Edit color contrast ratio to address accessibility concerns 

### ISSUE THREAD

**Issue by yiremorlans** [max-programming/hacktoberfest-projects#98]
# Accessibility bug
I came to the website to check out some issues for Hacktober Fest and noticed the contrast between the foreground text and background does not meet WCAG 2 AA minimum ratio thresholds for the following on the landing page:
- h1
- p
- span
- input

## Screenshots

<img width="641" alt="Screenshot 2023-09-24 at 11 15 37 AM" src="https://github.com/max-programming/hacktoberfest-projects/assets/93563580/df55cf9f-242c-457d-b7b9-fbd5fe1328fa">

### Current contrast ratio
The current WACAG 22 color contract ratio is **1.32 : 1**
<img width="1123" alt="Screenshot 2023-09-24 at 12 30 22 PM" src="https://github.com/max-programming/hacktoberfest-projects/assets/93563580/94584062-9f5a-4ba1-85c6-c7507daf06f8">

## WCAG2 Acceptance Criteria
- WCAG2 AA minimum is 4.5 : 1
- suggest WCAG2 AAA minimum is 4.5: 1 and 7 : 1

### Suggested solution for text color
<img width="1123" alt="Screenshot 2023-09-24 at 12 02 43 PM" src="https://github.com/max-programming/hacktoberfest-projects/assets/93563580/05a18b8d-ba57-42d0-9f29-6b6440581e38">

### Device:
Desktop
Browser: Chrome *latest version*

#### Comments
Additional contrast issues are found on the search results for projects but I think this would be a great first issue for some people and would make Hacktober Fest more inclusive for all people :)

**Comment by max-programming** [comment]
Thanks for the detailed information. Would you like to take this up?

**Comment by malik-vishu** [comment]
Can you assign me this issue? I will request PR on October 1st.

**Comment by max-programming** [comment]
Sure thing!

**Comment by malik-vishu** [comment]
I have to create a new branch right?

**Comment by malik-vishu** [comment]
@max-programming  **Is it all right?**

## Before
![finder_proj_before](https://github.com/max-programming/hacktoberfest-projects/assets/127117253/0745bad4-7c22-4f6a-bfb9-64498da8d37a)
***
![finder_proj_before2](https://github.com/max-programming/hacktoberfest-projects/assets/127117253/a4016ffd-9497-49e9-86d5-b784589c96ba)
***
![finder_proj_before3](https://github.com/max-programming/hacktoberfest-projects/assets/127117253/697fa87f-8b07-430d-993f-8b3499bb0fa2)
***
***
## After
![finder_proj_after](https://github.com/max-programming/hacktoberfest-projects/assets/127117253/83afee9d-a70e-4358-a4f6-a701345020ad)
***
![finder_proj_after2](https://github.com/max-programming/hacktoberfest-projects/assets/127117253/aa36e8cc-182d-4861-928c-e5af6ebb774f)
***
![finder_proj_after3](https://github.com/max-programming/hacktoberfest-projects/assets/127117253/60e21223-0e10-4361-b47c-77bccbe6f11c)
***
***

I completed the work, but I'd like it to be included in my hacktoberfest 2023 PR, so I'll make a request on October 1st. 
Also can you add hacktoberfest tag on this issue?
Thank You for the opportunity.

**Comment by Keshavraj52** [comment]
I hope this message finds you well! I'm eager to contribute more to our project. plz assign this issue to me
